### PR TITLE
add jocstaa to members

### PR DESCRIPTION
**Pull request by JOCSTAA**

### CODE REVIEW

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -179,6 +179,7 @@ orgs:
         - jlbutler
         - jlewi
         - jlyaoyuli
+        - jocstaa
```
Comment (connor-mccarthy) [inline github-orgs/kubeflow/org.yaml:182]: I think this should be capitalized

**Review by connor-mccarthy — CHANGES_REQUESTED** [review]

**Review by chensun — COMMENTED** [review]
/approve

**Pull request merged**